src/toolkit/matl_buy_policy.cc

```diff
@@ -55,34 +53,45 @@ void MatlBuyPolicy::set_throughput(double x) {
   throughput_ = x;
 }
 
-void MatlBuyPolicy::set_active(int x) {
-  assert(x > 0);
-  active_ = x;
+void MatlBuyPolicy::set_default_distributions() {
+  active_dist_ = new FixedIntDist(1);
+  dormant_dist_ = new FixedIntDist(-1);
+  size_dist_ = new FixedDoubleDist(1.0);
+  init_active_dormant();
```

Comment:
Since this gets called explicitly in cases where you don't use default distributions, I think it helps readability to call it explicitly after calling `set_default_distributions` in all the places where you do use the defaults.